packages/ui/src/components/StepsDialog.svelte

```diff
@@ -0,0 +1,271 @@
+      <div class="icon-wrapper">
+        {#if icon}<div class="wrapped-icon"><Icon {icon} size="medium" /></div>{/if}
+        <div class="title-wrapper">
+          {#if title}<span class="wrapped-title"><Label label={title} /></span>{/if}
+          {#if additionalStepInfo}<span class="wrapped-subtitle">{additionalStepInfo}</span>{/if}
+        </div>
+      </div>
+    </div>
+  </svelte:fragment>
+
+  <svelte:fragment slot="utils">
+    {@const isDone = currentStepIndex === steps.length - 1}
+
+    {#if currentStepIndex > 0}
+      <Button kind="secondary" label={ui.string.Back} on:click={async () => await handleStepSelect(currentStepIndex - 1)} />
+    {/if}
+    <Button
+      kind="primary"
+      label={isDone ? doneLabel : ui.string.Next}
+      disabled={!isStepValid}
+      loading={isSaving}
+      on:click={isDone ? handleDone : async () => await handleStepSelect(currentStepIndex + 1)}
+    />
+  </svelte:fragment>
+
+  <svelte:fragment slot="header">
+    {@const stepName = currentStep?.name}
+    {@const stepDescription = currentStep?.description}
+
+    <div class="header header-row between">
+      {#if stepName}<h4 class="no-margin"><Label label={stepName} /></h4>{/if}
+      {#if stepDescription}
+        <div class="buttons-group xsmall-gap" style:align-self="flex-start">
+          <span class="lines-limit-2"><Label label={stepDescription} /></span>
+        </div>
+      {/if}
+    </div>
+  </svelte:fragment>
+
+  <svelte:fragment slot="aside">
+    <Scroller>
+      <div class="default-padding">
+        {#if stepsName}<h4 class="no-margin"><Label label={stepsName} /></h4>{/if}
+        <ol>
+          {#each steps as step, index}
+            {@const selected = index === currentStepIndex}
+            {@const disabled = index > currentStepIndex && !(index === currentStepIndex + 1 && isStepValid)}
+            {@const fulfilled = index < currentStepIndex}
+
+            <li
+              class="overflow-label"
+              class:selected
+              class:disabled
+              class:fulfilled
+              title={step.name}
+              on:click={disabled || selected ? undefined : async () => await handleStepSelect(index)}
+            >
+              <Label label={step.name} />
+            </li>
+          {/each}
+        </ol>
+        {#if stepsDescription}
+          <span class="content-trans-color"><Label label={stepsDescription} /></span>
+        {/if}
+      </div>
+    </Scroller>
+  </svelte:fragment>
+
+  {#if currentStep?.component}
+    {@const { component, props } = currentStep}
+    {@const isMobile = $deviceInfo.isMobile}
+
+    <div
+      class="clear-mins"
+      class:popupPanel-body__mobile-content={isMobile}
+      class:popupPanel-body__main-content={!isMobile}
+      class:py-8={!isMobile}
+      class:max={!isMobile && useMaxWidth}
+    >
+      {#if typeof component === 'string'}
+        <Component is={component} {props} on:change={handleComponentChange} />
```

Comment:
Why not use parametrised slot? It will be more easy to use dialogs inplace with tabs content defined in same component.